src/domain/projects.rs

```diff
@@ -1,15 +1,22 @@
+use crate::extractors::time::json_time;
 use serde::{Deserialize, Serialize};
+use sqlx::types::chrono::NaiveDateTime;
 use sqlx::FromRow;
-
 #[derive(FromRow, Debug, Clone, PartialEq, Serialize)]
 pub struct Project {
     pub project_id: i32,
     pub main_document_id: i32,
-    pub owner: i32,
-    pub name: String,
+
+    pub owner_id: i32,
+    pub project_name: String,
+
+    #[serde(with = "json_time")]
+    pub created_at: NaiveDateTime,
+    #[serde(with = "json_time")]
+    pub last_modified: NaiveDateTime,
 }
 
-#[derive(FromRow, Debug, Clone, PartialEq, Deserialize)]
-pub struct ProjectMetadata {
+#[derive(sqlx::FromRow, Debug, Clone, PartialEq, Deserialize)]
```

Comment:
```suggestion
#[derive(FromRow, Debug, Clone, PartialEq, Deserialize)]
```

Reply:
rust likes to overcomplicate things sometimes, anyway i removed that